recipes-example/template-dbus-service/template-dbus-service.bb

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+DESCRIPTION = "Template for creating a D-Bus Service"
+HOMEPAGE = "https://github.com/Pelagicore/template-dbus-service"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+PV = "1.0+git${SRCREV}"
+PR = "r0"
+
+SRC_URI = "git@github.com:Pelagicore/template-dbus-service.git;protocol=ssh;branch=master"
+SRCREV = "f8a0e1bc987c2731b98ac69f55972ee9264ab518"
+
+inherit meson systemd
+
+DEPENDS += "glibmm gdbus-codegen-glibmm-native"
+
+S = "${WORKDIR}/git"
+
+# FIXME: Only include cmcli in -dev image?
+FILES_${PN} = " \
+    ${bindir}/cmcli \
+    ${bindir}/template-dbus-service \
+    ${datadir}/dbus-1 \
+    ${systemd_system_unitdir} \
+"
+
+SYSTEMD_SERVICE_${PN} = "template-dbus-service.service"
+# Activated through D-Bus (with systemd service) so set to disabled by default.
```

Comment:
kinda unnecessary comment

Reply:
If you know exactly what you are doing when you create a recipe it would have been unnecessary, but this template is supposed to be a help to someone creating a new D-Bus service, so I don't think it is unnecessary.

Reply:
Macro name is sort of self-explanatory. Also, there are docs.

Reply:
Just want to point out that the fact that this was needed and what the implications are if it is removed was not obvious. 

Will get warnings when package is installed with `rpm` due to no `systemctl enable` being invoked and there being no `[Install]` section in the `.service` file. There should not be one though if one only wants activation through D-Bus. Comment does not explain this... but perhaps it is too much details to go into.

Remove comment? Expand on it? Unnecessary to have this in all recipes (which is a risk if people just copy paste)? Do not know.

Reply:
I will remove this comment and instead add this information in the to-be-written documentation and instructions for the template-dbus-service.

Reply:
> Macro name is sort of self-explanatory. Also, there are docs.

@sashko : Sure, the macro is self-explanatory when it comes to the fact that the systemd service is disabled by default, but not that it will be activated through D-Bus. Or what do you mean?